import_export/resources.py

```diff
@@ -263,7 +264,11 @@ def __new__(cls, name, bases, attrs):
                     for option in dir(options)
                     if not option.startswith("_") and hasattr(options, option)
                 ]:
-                    setattr(meta, option, getattr(options, option))
+                    option_value = getattr(options, option)
```

Comment:
I notice that we have duplicated code from line 261-271 and 285-295.  This was there before your change, but how about we refactor into a single protected method on `DeclarativeMetaclass`?

Reply:
Okay. Let me look into it.

Reply:
Done. I am however having a challenge seeing what the conflicts are and resolving them. May you help me resolve them

Reply:
I have just merged the commits and pushed